src/optimade_maker/serve.py

```diff
@@ -119,13 +140,26 @@ def get_optimade_config(self):
             "debug": False,
             "insert_test_data": False,
             "insert_from_jsonl": str(jsonl_path.resolve()),
+            "create_default_index": True,
```

Comment:
This might be the one bit that is problematic @eimrek -- I added the ability to create a unique ID index to optimade-python-tools which is currently unreleased, when it gets released, `serve` will start making a unique index by default, which may conflict with any existing indexes you have... We could always set this to false by default to keep the old behaviour, and user's can now pass it as an env var anyway.